.github/workflows/tag-publish.yml

```diff
@@ -22,7 +22,7 @@ name: Bump, Tag, Publish, and Deploy
 on:
   push:
     branches:
-    - dev
+    - main
```

Comment:
You could put both dev and  main in the list for these (removing dev after the smoke clears), in case you had to rename it back for some reason.

Reply:
Excellent idea. I did that. Now I can pre-merge this and then do the branch switch.